packages/core/src/common/_react-transition.scss

```diff
@@ -4,12 +4,12 @@
 // and https://github.com/palantir/blueprint/blob/master/PATENTS
 
 /*
-A mixin to generate the classes for a React CSSTransitionGroup which animates any number of CSS properties at once.
+A mixin to generate the classes for a React CSSTransitionGroup which animates any number of CSS
+properties at once.
 
-Transitioned properties are specificed as a map of property names to lists of (inital value,
-final value). For enter & appear transitions, each property will transition from its initial to
-its final value. For leave transitions, each property will transition in reverse, from final to
-initial.
+Transitioned properties are specificed as a map of property names to lists of (inital value, final
+value). For enter & appear transitions, each property will transition from its initial to its final
+value. For leave transitions, each property will transition in reverse, from final to initial.
```

Comment:
would be nice to rewrap all comments too... I understand it's tedious but there's tooling out there to help with this, for example https://marketplace.visualstudio.com/items?itemName=stkb.rewrap